docs/user/contributing.rst

```diff
@@ -110,6 +111,23 @@ Mypy won't fix your code for you, but will warn you about potential issues with
 
 If you've never used mypy before or aren't familiar with `Python type hints <https://docs.python.org/3/library/typing.html>`_, this check can be particularly daunting. Don't hesitate to ask for help with resolving any of these warnings on your pull request.
 
+Prettier is a code formatter that helps to ensure consistent code style across a project. It supports various languages. Follow these steps to install Prettier:
+
+1. Install Node.js: Prettier is a Node.js module, so you need to have Node.js installed on your system. You can download and install Node.js from the `Node.js official website <https://nodejs.org/en>`_.
+2. Install Prettier: Use the following command to install the Prettier module in your project:
+
+.. code-block:: console
+
+   $ npm install requirements/
```

Comment:
```suggestion
   $ npm install prettier
```
Users shouldn't know about `requirements`

Reply:
So here we will use `npm install prettier --no-save` and will keep `/*.json` in `.gitignore` for case is someone would mess up command or something like that. You agree with this ?

Reply:
Yep, sounds good to me